cmd/cli/kubectl-kyverno/commands/test/test.go

```diff
@@ -278,6 +284,17 @@ func runTest(out io.Writer, testCase test.TestCase, registryAccess bool) (*TestR
 		resourceKey := generateResourceKey(resource)
 		engineResponses = append(engineResponses, ers...)
 		testResponse.Trigger[resourceKey] = ers
+
+		for _, res := range ers {
+			if res.Policy().AsMutatingAdmissionPolicy() != nil {
+				for i := range testCase.Test.Results {
+					// match the policy name; ignore rule name for MAP/MAPB
+					if testCase.Test.Results[i].Policy == res.Policy().GetName() {
+						testCase.Test.Results[i].IsMutatingAdmissionPolicy = true
+					}
+				}
+			}
+		}
```

Comment:
We don't need to set the field `isMutatingAdmissionPolicy` to true in the codebase. The user should set this field in case the policy is of type MAP. Therefore, there is no need for this code.